source/backend/.s2i/environment

```diff
@@ -1 +0,0 @@
-DOTNET_STARTUP_PROJECT=api/Pims.Api.csproj
```

Comment:
I think this file is used as part of openshift building .NET
Did you test this change in openshift?